test/tests/haproxy-basics/haproxy.cfg

```diff
@@ -8,5 +8,3 @@ frontend http-in
 	default_backend google
 backend google
 	server google0 www.google.com
-	# this doesn't work on HAProxy 1.4 :(
-	#http-request replace-header Host .* www.google.com
```

Comment:
Another option would be to simply uncomment this now that 1.4 is not supported and then not do anything special with `curl` -- just hit our local engine directly.